src/masonite/providers/HelpersProvider.py

```diff
@@ -17,6 +17,7 @@ def boot(self):
         self.application.make("view").share(
             {
                 "request": lambda: request,
+                "session": lambda: request.app.make("session"),
                 "auth": request.user,
```

Comment:
Just noticed but does this also need to be wrapped in a lambda?

Reply:
I am not sure... maybe to have an up to date session instance ? 
Maybe it's not needed and in a view we could simply use `session.get("key")ˋ